app/styles/layout/_aside.scss

```diff
@@ -4,31 +4,90 @@ aside {
   background-color: $grey-lighter;
   color: $black;
   height: auto;
-  overflow-y: scroll; // TODO idk if I should do this? is this ok to do?
+  overflow-y: auto;
```

Comment:
Just a quick explanation, setting it to `scroll` results in the scrollbar appearing even when not needed (being disabled).

By setting it to `auto`, the scrollbar will show up only when the content actually overflows the container.